interactive_engine/compiler/src/main/java/com/alibaba/graphscope/gremlin/plugin/processor/IrStandardOpProcessor.java

```diff
@@ -329,17 +337,36 @@ protected void processTraversal(
                 PegasusClient.JobConfig.newBuilder()
                         .setJobId(jobId)
                         .setJobName(jobName)
-                        .setWorkers(PegasusConfig.PEGASUS_WORKER_NUM.get(configs))
-                        .setBatchSize(PegasusConfig.PEGASUS_BATCH_SIZE.get(configs))
-                        .setMemoryLimit(PegasusConfig.PEGASUS_MEMORY_LIMIT.get(configs))
-                        .setBatchCapacity(PegasusConfig.PEGASUS_OUTPUT_CAPACITY.get(configs))
+                        .setWorkers(PegasusConfig.PEGASUS_WORKER_NUM.get(queryConfigs))
```

Comment:
Will the `configs` value be preserved?
Consider this scenario:
1. The default PegasusConfig.PEGASUS_WORKER_NUM = 4 if not specified
2. Ir Server is created with config pegasus.worker.num = 2
3. Specify pegasus.worker.num = 1 by `g.with(xxx,xx)`
The latter settings should take higher priority than previous setting
Then, the following conditions should be met:
- with 1, worker-num = 4
- with 1,2 worker-num = 2
- with 1,2,3, or 1, 3  worker-num = 1

Reply:
![Uploading image.png…]()

Reply:
![Uploading image.png…]()

Reply:
make pegasus.XX settings configurable per query:
```
g.with(pegasus.worker.num).V()
g.with(pegasus.batch.size).V()
g.with(pegasus.output.capacity).V()
```
the priority is : query > system env > system property > config file